src/components/globalnavigasjon/GlobalNavigasjon.tsx

```diff
@@ -33,83 +34,60 @@ export enum Menypunkter {
   HISTORIKK = "HISTORIKK",
   VEDTAK = "VEDTAK",
   ARBEIDSUFORHET = "ARBEIDSUFORHET",
+  FRISKTILARBEID = "FRISKTILARBEID",
 }
 
-export type Menypunkt = { navn: string; sti: string; menypunkt: Menypunkter };
-
-const nokkelinformasjonMenypunkt = {
-  navn: "Nøkkelinformasjon",
-  sti: "nokkelinformasjon",
-  menypunkt: Menypunkter.NOKKELINFORMASJON,
-};
-
-const aktivitetskravMenypunkt = {
-  navn: "Aktivitetskrav",
-  sti: "aktivitetskrav",
-  menypunkt: Menypunkter.AKTIVITETSKRAV,
-};
-
-const behandlerdialogMenypunkt = {
-  navn: "Dialog med behandler",
-  sti: "behandlerdialog",
-  menypunkt: Menypunkter.BEHANDLERDIALOG,
-};
-
-const historikkMenypunkt = {
-  navn: "Logg",
-  sti: "logg",
-  menypunkt: Menypunkter.HISTORIKK,
-};
-
-const motemodulMenypunkt = {
-  navn: "Dialogmøter",
-  sti: "moteoversikt",
-  menypunkt: Menypunkter.DIALOGMOTE,
-};
-
-const sykmeldingerMenypunkt = {
-  navn: "Sykmeldinger",
-  sti: "sykmeldinger",
-  menypunkt: Menypunkter.SYKMELDINGER,
+export type Menypunkt = { navn: string; sti: string };
+
+const allMenypunkter: {
```

Comment:
Litt refaktorering sånn at koden ikke kompilerer om man legger til en ny enum-verdi i `Menypunkter` uten å definere `navn` og `sti` for menypunktet her.